### PR TITLE
Add UBI container build

### PR DESCRIPTION
**Pull request by major**

### CODE REVIEW

**Comment by major** [comment]
Looks like the action ran fine over in my fork: https://github.com/major/osbuild-installer/runs/1068595389?check_suite_focus=true

File: .github/workflows/container.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+      - name: Verify that the startup log line is in the logs
+        run: |
+          docker logs osbuild-installer | grep "Starting osbuild-installer server"
+
+      - name: Check that the API is responding
+        run: curl -sv http://127.0.0.1:8086/api/osbuild-installer/v1/version
+
+      - name: Stop the container
+        run: docker stop --time 1 osbuild-installer
```
Comment (Gundersanne) [inline .github/workflows/container.yml:32]: Could you add a single newline here at the end of the file?

File: distribution/Dockerfile-ubi [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+COPY . .
+ENV GOFLAGS=-mod=vendor
+# RUN go get -d -v ./...
+RUN go install -v ./...
+
+FROM registry.access.redhat.com/ubi8/ubi:latest
+RUN mkdir /app
+COPY --from=builder /root/go/bin/osbuild-installer /app/
+EXPOSE 8086
+CMD ["/app/osbuild-installer", "-v"]
```
Comment (Gundersanne) [inline distribution/Dockerfile-ubi:13]: Same here

**Review by Gundersanne — CHANGES_REQUESTED** [review]
excellent! just one nitpick :p

File: .github/workflows/container.yml (hunk 1 above)
Comment (major) [reply .github/workflows/container.yml]: Done!

File: distribution/Dockerfile-ubi (hunk 2 above)
Comment (major) [reply distribution/Dockerfile-ubi]: Done!

**Review by Gundersanne — APPROVED** [review]

**Pull request merged**